README.md

```diff
@@ -112,6 +112,11 @@ repositories:
 ```
 _Note: The `summary` and `introduction` fields are to be rendered as markdown, allowing for bullet points, links and other simple formatting._
 
+### Best Practices
+To have github issues shown in the site, it has to have any of the label stated in the project's yaml file. To have bugzilla bugs to appear in the site, 
+it has to have good-first-bug as keywords. For both bugzilla bugs and github issues, we only display unassigned bugs / issues by default. You can
+view the assigned bugs / issues by clicking on the filter icon besides the Feeling Adventurous Button.
+
```

Comment:
### Best Practices

Below is information about how to get a bug or issue appear on the site.

#### GitHub

Codetribute will read the `repositories` field from the yaml file and will take extract the issues that match the repository labels.

_Example: Display servo issues tagged with the label *E-easy*_

```yaml
repositories: 
 - servo/servo: E-easy
```

_Example: Display telemetry-dashboard issues tagged with either the *mentored* or *good first issue* label_

```yaml
repositories:
 - mozilla/telemetry-dashboard: ['mentored', 'good first issue']
```

#### Bugzilla

Codetribute will read the `products` field from the yaml file and will extract bugs with keyword `good-first-bug`. Here are some ways to display bugs on the site.

_Example: Display all Taskcluster bugs with keyword *good-first-bug*_

```yaml
products: 
 - Taskcluster
```

_Example: Display Taskcluster bugs that are under the Tools component with keyword *good-first-bug*_

```yaml
products:
 - Taskcluster: ['Tools']
```

Reply:
💯 shall we add that we only display unassigned ones too?

Reply:
We could, but I don't expect someone to worry if an assigned issue/bug is not showing up. I'm okay not mentioning it.